airbyte_cdk/sources/declarative/retrievers/async_retriever.py

```diff
@@ -74,14 +78,36 @@ def _validate_and_get_stream_slice_jobs(
         """
         return stream_slice.extra_fields.get("jobs", []) if stream_slice else []
 
+    def _get_cursor_slice_info(
```

Comment:
nit: I think it's better to have just `_get_cursor_slice`, as we return `cursor_slice` as is and don't add any other information related to it

Reply:
Thanks! I've changed it as suggested.